mappings/net/minecraft/network/message/SentMessage.mapping

```diff
@@ -0,0 +1,29 @@
+CLASS net/minecraft/class_7604 net/minecraft/network/message/SentMessage
+	METHOD method_44852 getOriginalMessage ()Lnet/minecraft/class_7471;
+	METHOD method_44853 toPacket (Lnet/minecraft/class_3222;Lnet/minecraft/class_2556$class_7602;)Lnet/minecraft/class_7438;
+		ARG 1 player
+		ARG 2 params
+	METHOD method_44854 of (Lnet/minecraft/class_5837;Lnet/minecraft/class_7436;)Lnet/minecraft/class_5837;
+		ARG 0 message
+		ARG 1 profile
+	METHOD method_44855 (Lnet/minecraft/class_5837;Lnet/minecraft/class_7436;Lnet/minecraft/class_7471;)Lnet/minecraft/class_7604;
+		ARG 2 rawMessage
+	METHOD method_44856 afterPacketsSent (Lnet/minecraft/class_3324;)V
```

Comment:
```suggestion
	METHOD method_44856 onAfterPacketsSent (Lnet/minecraft/class_3324;)V
```

Reply:
Hmm, this isn't FAPI, and I don't think the on prefix makes sense.

Reply:
Okay, I think the name is fine then.